spec/selenium_compatibility_spec.rb

```diff
@@ -20,12 +24,14 @@
             log.push(event.target.id + '.' + event.type);
           };
           var elements = document.getElementsByTagName("input");
+          var selectTag = document.getElementById("five");
```

Comment:
Can we add `selectTag` to `elements` somehow so it doesn't need to be treated specially below? For example, we could add a `watch` class to the inputs we want to watch, and use `getElementsByClassName` above.